ui/lib/pki/addon/templates/configuration/edit.hbs

```diff
@@ -1 +1,13 @@
-configuration.edit
```

Comment:
very cool to see this part of the project getting tackled. It's a big step towards completion.